components/shared/profile/edit/EditProfile.tsx

```diff
@@ -226,12 +226,14 @@ const EditProfile = ({ user }: EditProfileProps) => {
         </label>
         <div className="flex  w-full shrink flex-col">
           {technologies?.map((tech: any, index) => {
+            const icon = techStackBadges.find((badge) => badge.name === tech);
             return (
               <div
                 key={index}
                 className="paragraph-3-bold order-2 flex h-7 w-1/2 flex-col items-center justify-start bg-black-600 p-3 text-white-100"
               >
                 {" "}
+                {icon.icon()}
```

Comment:
```suggestion
                {icon.icon}
```

Reply:
same with 'result.icon()' further down the file.

Reply:
Your css is still causing issues here. 
```js
<div
                key={index}
                className="paragraph-3-bold order-2 flex w-1/2 items-center justify-start bg-black-600 p-3 text-white-100"
              >
```

Reply:
yep i am working on functionality, and going to re-do the styling when i have things hooked up properly, so i can focus on straight functionality and then focus on straight styling for hopefully a steadier workflow